intellij-toml/core/src/main/kotlin/org/toml/ide/annotator/TomlAnnotator.kt

```diff
@@ -24,5 +24,10 @@ class TomlAnnotator : AnnotatorBase() {
                 holder.newAnnotation(HighlightSeverity.ERROR, "Inline tables are intended to appear on a single line").create()
             }
         }
+
+        val parent = element.parent
+        if (element.text == "," && parent is TomlInlineTable && element.textOffset > parent.entries.last().textOffset) {
```

Comment:
Could you please check the element type instead of its text? `psi.text` looks very suspicious and almost always there is a better way to do things.